File: 8KYU/Playing with cubes II/solution.cs
using System;

public class Cube
{
  private int _side;
  
  //This cube needs your help. 
  //Define a constructor which takes one integer and assignes its value to 'Side'
  public Cube() => _side = 0;
  
  public Cube(int s) => _side = (int)Math.Abs(s);
  
  public int GetSide() => _side;
  
  public void SetSide(int s) => _side = (int)Math.Abs(s);
}

File: 8KYU/Playing with cubes II/solution.py
class Cube(object):
      
    def __init__(self, length=0):
        self._side = abs(length)
        
    def get_side(self):
        """Return the side of the Cube"""
        return self._side

    def set_side(self, new_side):
        """Set the value of the Cube's side."""
        self._side = abs(new_side)